### DISCUSSION TITLE
THANK YOU!!! Would love links to Spanish NWT

### DISCUSSION THREAD

**Discussion by SanSon97** [majal/bible-linker-google-docs#21] [Ideas]
Dear Majal, my name is Santi. Although I love computing, software and all other Tech topics, I don't know anything about coding. However, I found your repository and installed it in Google Docs. Wonderful! Me and another friend have been trying to create something similar in Word with Macros using VBA with no success.

I just want to thank you and request you, if possible, to add the Spanish NWT to the plug in opening the links to JWLibrary.

Take care! . (Sorry, this was the only way I found to contact you ☺)

**Comment by majal** [comment]
Hi @SanSon97!

Thank you for making this request. I initially made the script for myself, but thought it could be helpful to others too. Your request tells me that adding languages will make it even more useful. 

There was [a pull request (PR) to add the Tagalog language](https://github.com/majal/bible-linker-google-docs/pull/20). In the discussion, we were looking to refactor the code and make it modular.

What could help Santi is if you can refer coders who can help in refactoring the code. They can collaborate with @JerwellSavella and @venjot. Once the new code is set, it will be easier to add new languages to the script.

Jehovah’s blessings!

**Comment by majal** [comment]
https://github.com/majal/bible-linker-google-docs/discussions/24#discussioncomment-5553877
